packages/svelte/tests/runtime-runes/samples/event-store-no-hoisting/main.svelte

```diff
@@ -5,8 +5,7 @@
 
 	function setStore() {
 		store = writable(0, () => {
-			console.log('start');
-			return () => console.log('stop');
+			return () => {};
```

Comment:
Why did this change?

Reply:
drive-by. They show up as annoying and unnecessary console logs in the test output (the corresponding test config doesn't use `logs`)